src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -100,6 +100,16 @@ export class DashboardComponent implements OnDestroy {
         filter: false,
         renderComponent: JobStatusIconRenderComponent,
       },
+      no_of_connectors_executed: {
```

Comment:
Have you added a `SerializerMethodField` in the `JobListSerializer` for this ? similar to [this](https://github.com/intelowlproject/IntelOwl/blob/8f14384c315613567d39b3c5aa612d61f05c3b91/api_app/serializers.py#L66)

Reply:
https://github.com/orgs/intelowlproject/projects/2#card-64203186